app.js

```diff
@@ -18,11 +18,12 @@ changeColor();
 function dateAndTime() {
     const dateAndTime = document.querySelector('header p');
     let time = new Date();
+    let month = "October";
```

Comment:
This should be dynamic, use javascript to get the current month

Reply:
If i get the month, it will be showing as number

Reply:
Then use the number to get the month

Reply:
ok sir